.github/workflows/solidity.yml

```diff
@@ -217,6 +217,11 @@ jobs:
       - name: Setup NodeJS
         uses: ./.github/actions/setup-nodejs
 
+      - name: Install Foundry
+        uses: ./chainlink/.github/actions/install-solidity-foundry
```

Comment:
This pathing is confusing me.

Working directory is `chainlink/contracts`, but then the path to the action is relative to that in `/chainlink/.github/actions/install-solidity-foundry`?

Reply:
Copypaste fail from a previous job. Good catch.

Fixed in: b2f412c70ccdb879d8fc79484530f9745ff4248e